ui/src/composables/private/__tests__/use-field.cy.js

```diff
@@ -133,102 +209,206 @@ describe('use-field API', () => {
         })
       })
 
       describe('(prop): suffix', () => {
-        it.skip(' ', () => {
-          //
+        it('should display a suffix', () => {
+          const suffix = 'Hello there!'
+          mountQFieldWrapper({
+            props: {
+              suffix
+            }
+          })
+
+          getHostElement().get('.q-field__suffix').should('exist').should('contain', suffix)
         })
       })
 
       describe('(prop): clear-icon', () => {
-        it.skip(' ', () => {
-          //
+        it('should display custom clear-icon when one is set', () => {
+          const model = ref(selectOptions[ 0 ])
+          const clearIcon = 'custom-clear-icon'
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options: selectOptions,
+              clearable: true,
+              clearIcon
+            }
+          })
+
+          getHostElement().get('.q-field__append').get('button').should('contain', clearIcon)
         })
       })
 
       describe('(prop): label-slot', () => {
-        it.skip(' ', () => {
-          //
+        it('should force the use of the label slot even when a label prop is set', () => {
+          const model = ref(selectOptions[ 0 ])
+          const labelSlot = 'Hello world'
+
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options: selectOptions,
+              labelSlot: true,
+              label: 'Hello there'
+            },
+            slots: {
+              label: () => labelSlot
+            }
+          })
+
+          getHostElement().find('.q-field__label').should('contain.text', labelSlot)
         })
       })
 
       describe('(prop): bottom-slots', () => {
-        it.skip(' ', () => {
-          //
+        it('should use a bottom error slot', () => {
+          const model = ref(selectOptions[ 0 ])
+          const bottomSlot = 'Hello there'
+
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options: selectOptions,
+              bottomSlotSlots: true,
+              error: true
+            },
+            slots: {
+              error: () => bottomSlot
+            }
+          })
+
+          getHostElement().find('.q-field__bottom')
+            .should('contain.text', bottomSlot)
+        })
+
+        it('should use a bottom hint slot', () => {
+          const model = ref(selectOptions[ 0 ])
+          const bottomSlot = 'Hello there'
+
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options: selectOptions,
+              bottomSlots: true
+            },
+            slots: {
+              hint: () => bottomSlot
+            }
+          })
+
+          getHostElement().find('.q-field__bottom').should('contain.text', bottomSlot)
+        })
+
+        it('should use a bottom counter slot', () => {
```

Comment:
Isn't this test the same as the previous one?

Reply:
Fixed. This one should using the `counter` prop instead of the `hint` slot as the previous one.